tests/Integration/Infrastructure/Controller/MyArea/Organization/VisaModel/EditVisaModelControllerTest.php

```diff
@@ -31,12 +31,12 @@ private function visaModels(): array
     public function testEdit(string $name, ?string $description, array $visas): void
     {
         $client = $this->login('mathieu.fernandez@beta.gouv.fr');
-        $crawler = $client->request('GET', '/mon-espace/organizations/' . OrganizationFixture::MAIN_ORG_ID . '/visa_models/7eca6579-c07e-4e8e-8f10-fda610d7ee73/edit');
+        $crawler = $client->request('GET', '/mon-espace/organizations/' . OrganizationFixture::MAIN_ORG_ID . '/visa_models/65c12316-e210-445d-9169-0298b13b3b30/edit');
 
         $this->assertResponseStatusCodeSame(200);
         $this->assertSecurityHeaders();
-        $this->assertSame('Réglementation de vitesse en agglomération', $crawler->filter('h2')->text());
-        $this->assertMetaTitle('Réglementation de vitesse en agglomération - DiaLog', $crawler);
+        $this->assertSame('Interdication de circulation', $crawler->filter('h2')->text());
```

Comment:
Typo dans la fixture ^^

```suggestion
        $this->assertSame('Interdiction de circulation', $crawler->filter('h2')->text());
```